keyboards/rii/518bt/keymaps/default/keymap.c

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2025 tim-eastwood
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#define CTALDEL LCTL(LALT(KC_DEL))
+
+void keyboard_pre_init_user(void) {
+  gpio_set_pin_output(B2);
+  gpio_set_pin_output(B10);
+  gpio_set_pin_output(B11);
+  gpio_write_pin_low(B2);
+  gpio_write_pin_low(B10);
+  gpio_write_pin_low(B11);
```

Comment:
What are these for?

Reply:
the PCB has 4 LEDs - one for capslock, the other three for whatever. If the pins are left to float they glow slightly

Reply:
It would be better to do this at keyboard level then, and make the pin names into defines to avoid magic numbers and allow any keymap to turn the LEDs on and off - you may also want to make helper functions for this; have a look at `woodpad` for example.